sdk/monitor/monitor-query-logs/sample.env

```diff
@@ -0,0 +1,21 @@
+# Retrieve these values from an instance in the Azure Portal.
+MONITOR_WORKSPACE_ID=<The "Workspace ID", as given in the Azure portal for your Log Analytics resource>
+METRICS_RESOURCE_ID=<An Azure resource ID to query for metrics. Ex: /subscriptions/<full resource path>>
+
+# Used by tests and samples to populate data
+APPLICATIONINSIGHTS_CONNECTION_STRING=<Azure Application Insights connection string>
```

Comment:
I don't see this being used in samples, should it be deleted?

Reply:
This was marked as resolved without being addressed nor a comment so unresolving.

Reply:
missed updating here, I removed the file and brought it back, as the sample validation step was failing due to a missing sample.env file.